src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void register_nullUser_notOk() {
+        User nullUser = new User(null, null, null);
+        assertThrows(RuntimeException.class, () ->
+                registrationService.register(nullUser));
+    }
+
+    @Test
+    void register_nullUserLogin_notOk() {
+        User nullUserLogin = new User(null, "password", 18);
+        assertThrows(RuntimeException.class, () ->
+                registrationService.register(nullUserLogin));
+    }
+
+    @Test
+    void register_shortUserLogin_notOk() {
+        User shortUserLogin = new User(null, "password", 18);
+        assertThrows(RuntimeException.class, () ->
+                registrationService.register(shortUserLogin));
+    }
+
+    @Test
+    void register_nullUserPassword_notOk() {
+        User nullUserPassword = new User("login", null, 18);
+        assertThrows(RuntimeException.class, () ->
+                registrationService.register(nullUserPassword));
+    }
+
+    @Test
+    void register_shortUserPassword_notOk() {
+        User shortUserPassword = new User("login", null, 18);
```

Comment:
The `shortUserPassword` User object is being created with a `null` password instead of a short password. Provide a short password to match the test case description.